extensions/transform/org.eclipse.smarthome.transform.scale/README.md

```diff
@@ -1,16 +1,51 @@
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
 
-The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
+The file is expected to exist in the `transform` configuration directory and its ending has to be `.scale`.
+It should follow the format given in the table below.
+
+Range expressions always contain two parts.
+The range to scale on, which is located left from the equality sign and the corresponding output string on the right of it.
+A range consists of two bounds. Both are optional, the range is then open. Both bounds can be inclusive or exclusive.
+
+| Scale Expression | Returns XYZ when the given value is                        |
+|------------------|------------------------------------------------------------|
+| `[12..23]=XYZ`   | `between (or equal to) 12 and 23`                          |
+| `]12..23[=XYZ`   | `between 12 and 23 (12 and 23 are excluded in this case.)` |
+| `[..23]=XYZ`     | `lower than or equal to 23`                                |
+| `]12..]=XYZ`     | `greater than 12`                                          |
+
+These expressions are evaluated from top to bottom.
+The first range that includes the value is selected.
 
 ## Example
 
-transform/temps.scale:
+The following example shows how to break down numeric UV values into fixed UV index categories.
+We have an example UV sensor that sends numeric values from `0` to `100`, which we then want to scale into the [UV Index](https://en.wikipedia.org/wiki/Ultraviolet_index) range.
 
-```properties
-[12..23]=Temp between 12 or higher and 23 or lower
-]12..23[=Temp over 12 and below 23
-..23]=Any temp up to and including 23
-]12..=Any temp above 12
+Example item:
+
+```java
+Number Uv_Sensor_Level "UV Level [SCALE(uvindex.scale):%s]"
 ```
+
+Referenced scale file `uvindex.scale` in the `transform` folder:
+
+```python
+[..3]=1
+]3..6]=2
+]6..8]=3
+]8..10]=4
+[10..100]=5
+```
+
+Each value the item receives, will be categorized to one of the five given ranges.
+Values **lower than or equal to 3** are caught with `[..3]=1`.
+Greater values are catched in ranges with 2 values as criteria.
```

Comment:
~~are catched~~ -> "are **caught**" (or maybe rather "are **matched**")

Reply:
I think "matched" it will be, when I compare the options.
That sounds/reads the best. (My opinion)